lambda/src/index.js

```diff
@@ -91,30 +85,28 @@ EchoSonos.prototype.intentHandlers = {
     PandoraMusicIntent: function (intent, session, response) {
         console.log("PandoraMusicIntent received");
         loadCurrentRoomAndService('DefaultEcho', intent.slots.Room.value, function(room, service) {
-            pandoraHandler(room, '/play/', intent.slots.Name.value, response);
+            pandoraHandler(room, 'pandoraPlay', intent.slots.Name.value, response);
         });
     },
 
     PandoraThumbsUpIntent: function (intent, session, response) {
         console.log("PandoraThumbsUpIntent received");
         loadCurrentRoomAndService('DefaultEcho', intent.slots.Room.value, function(room, service) {
-            pandoraHandler(room, '/thumbsup', '', response);
+            pandoraHandler(room, 'pandoraThumbsUp', '', response);
         });
     },
 
     PandoraThumbsDownIntent: function (intent, session, response) {
         console.log("PandoraThumbsDownIntent received");
         loadCurrentRoomAndService('DefaultEcho', intent.slots.Room.value, function(room, service) {
-            pandoraHandler(room, '/thumbsdown', '', response);
+            pandoraHandler(room, 'pandoraThumbsDown', '', response);
         });
     },
 
     PlayPresetIntent: function (intent, session, response) {
         console.log("PlayPresetIntent received");
-        options.path = '/preset/' + encodeURIComponent(intent.slots.Preset.value.toLowerCase());
-        httpreq(options, function(error) {
-            genericResponse(error, response);
-        });
+        var promise = sonosProxy.preset();
```

Comment:
argument ist missing: needs to be :
`var promise = sonosProxy.preset(intent.slot.Preset.value);`

Since this is the intent used in the testcase, this could otherwise frustrate people trying it out.